covfee/client/tasks/continuous_annotation/index.tsx

```diff
@@ -305,14 +305,23 @@ const ContinuousAnnotationTask: React.FC<Props> = (props) => {
   //       does change, which trigger a change in the sources, i.e., the video url
   //       being used.
   const videoPlayerOptions = useMemo(() => {
+    let participant = ""
+    let source = { ...props.spec.media[selectedCamViewIndex] }
+    if (validAnnotationsDataAndSelection) {
+      // Note: consider optimizing such that if replaceable strings are not
+      //       found, then it fallback into not having selectedAnnotationIndex
+      //       as a dependency, and thus avoiding the videos from reload.
+      participant = annotationsDataMirror[selectedAnnotationIndex].participant
+      source.src = source.src.replace("{participant}", participant)
```

Comment:
This is only trigger on participant selection change?

Reply:
As per the useMemo dependency list, it triggers every time selectedAnnotationIndex changes. Either when a different participant is selected by the user, or whenever the annotation type changes (like changing from speaking to laughing) in case the current instance supported multiple